spec/liquid_tags/twitch_tag_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+require "nokogiri"
+
+RSpec.describe TwitchTag, type: :liquid_tag do
+  let(:slug) { "CuteSpicyNostrilDoritosChip" }
+
+  def assert_parses(slug, token)
+    liquid = Liquid::Template.parse("{% twitch #{token} %}").render
+    expect(liquid).to include "https://clips.twitch.tv/embed?autoplay=false&clip=#{slug}&parent=localhost"
+  end
+
+  it "accepts twitch clip slug" do
+    assert_parses slug, slug
+  end
+
+  it "accepts twitch clip slug with wonky whitespace" do
+    assert_parses slug, " #{slug}  \t"
+  end
+
+  it "forbids inserting autoplay option" do
+    assert_parses slug, "#{slug}&autoplay=true"
+  end
```

Comment:
this test is not very self explanatory. "assert it parses the slug if you pass the slug with autoplay true" but maybe we should test "make sure the slug doesn't contain autoplay=true"